core/src/main/java/com/sequenceiq/cloudbreak/service/loadbalancer/LoadBalancerConfigService.java

```diff
@@ -272,8 +272,8 @@ private boolean shouldCreateOutboundLoadBalancer(boolean createExternalLb, Stack
                 && LoadBalancerSku.STANDARD.equals(sku)
                 && !Optional.of(network)
                             .map(EnvironmentNetworkResponse::getAzure)
-                            .map(EnvironmentNetworkAzureParams::isNoOutboundLoadBalancer)
-                            .orElse(false);
+                            .map(EnvironmentNetworkAzureParams::getNoOutboundLoadBalancer)
+                            .orElse(network.isExistingNetwork());
```

Comment:
Few questions here
1. What does getNoOutboundLoadBalancer return when the environment is created using UI?
2. When isExistingNetwork is false, we should create outbound lb.

Reply:
1) We cannot save null in the database to keep retro compatibility. The value committed depends on the existence or not of the network. 
2) Yes if the customer doesn't send the noOutboundLoadBalancer flag.